src/bcf/report/oncoprint.rs

```diff
@@ -368,7 +369,17 @@ pub fn oncoprint(
                         rec.extend(record_tuples.iter().map(|(r, _)| r.clone()));
                     }
                     1 => rec.extend(filter_canonical.iter().map(|(r, _)| r.clone())),
-                    _ => panic!("Found more than one variant annotated as canonical!"),
+                    _ => {
+                        rec.extend(filter_canonical.iter().map(|(r, _)| {
+                            dbg!(&r);
+                            r.clone()
+                        }));
+                        let alterations = filter_canonical
+                            .iter()
+                            .map(|(r, _)| r.alteration.clone())
+                            .collect_vec();
+                        warn!("Found more than one variant of gene {} annotated as canonical! The alterations marked as canonical are: {:?}", &k, &alterations);
```

Comment:
```suggestion
                        warn!("Found more than one transcript of gene {} annotated as canonical! The transcripts marked as canonical are: {:?}", &k, &transcripts);
```

Naming: there are multiple transcripts marked as canonical, not alterations.